pypeclub/OpenPype
# Flame - create publishable clips

## Brief description
Flame converting timeline segments to publishable instances

## Description
First step towards publishing workflow in Flame. This feature will allow user to convert selected timeline segments into publishable instances for farther processing. Segments are sequentially renamed on demand and visually tagged.

## Testing notes:
1. Open Flame project from openpype launcher
2. open your testing sequence
3. select any clip on timeline
4. right button click and go to Openpype/Create
5. Hit accept on Create publishable clips
6. creator gui will popup 
7. configure hierarchy and name convention and hit accept
8. clips should be marked in middle with openpypedata marker with json text in comment
